III.tdd-katas/calculator-tdd/add.test.js

```diff
@@ -0,0 +1,40 @@
+const add = require("./add");
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("")).toBe(0);
+    });
+    test('should return 1 for "1" string passed to add', () => {
+        expect(add("1")).toBe(1);
+    });
+    test('should return 3 for "1,2" string passed to add', () => {
+        expect(add("1,2")).toBe(3);
+    });
+});
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
```

Comment:
The test title here does not match the content of the test - I think this is a copy-paste error from above.